.gitmodules

```
@@ -42,3 +42,6 @@
 	path = third-party/build-deps
 	url = https://github.com/LizardByte/build-deps.git
 	branch = dist
+[submodule "third-party/nlohmann_json"]
+	path = third-party/nlohmann_json
+	url = https://github.com/nlohmann/json
```

```suggestion
[submodule "third-party/nlohmann_json"]
	path = third-party/nlohmann_json
	url = https://github.com/nlohmann/json
	branch = master
```

You'll also need to checkout this submodule branch and push the changes back to your PR branch.